   ml_ops/sm-datazone_import/cross-account-setup.md

```diff
@@ -0,0 +1,59 @@
+### Cross Account Setup
+In this scenario, we will use two accounts, a parent account hosting the DataZone domain, and a child account that contains
+a SageMaker Domain nd UserProfile that we would like to link, and import into the DataZone domain. Here AccountA is the parent
+account and AccountB is the child account.
+
+1. **[In the Parent Account]** Create a DataZone domain (make sure you are NOT using the Unified UI). It should say “Create a DataZone Domain”.
```

Comment:
sigh ya this complicates the instructions

Reply:
Yea. We need to clarify specifically where the entry point is for the legacy UI.